frontend/src/metabase/dashboard/components/DashboardEmbedHeaderButton/DashboardEmbedHeaderButton.tsx

```diff
@@ -48,13 +48,15 @@ export const DashboardEmbedHeaderButton = forwardRef(
         }
         offset={8}
       >
-        <DashboardHeaderButton
-          data-testid="dashboard-embed-button"
-          icon="share"
-          disabled={disabled}
-          onClick={onClick}
-          ref={ref}
-        />
+        <Box>
```

Comment:
What does `Box` do in this case? Is there any visual change?

Reply:
So basically it was an issue with the fact that Mantine tooltip, by default, doesn't render when the button is disabled. But, I just saw on the mantine docs (in the Button section for some reason, not the tooltip) that we can use the `data-disabled` prop to allow tooltips to work on disabled buttons (https://v6.mantine.dev/core/button/#disabled-button-with-tooltip). I'll use this instead

Reply:
Gotcha